music21/test/testSingleCoreAll.py

```diff
@@ -122,7 +129,7 @@ def main(testGroup=('test',), restoreEnvironmentDefaults=False, limit=None, verb
 def ciMain():
     # the main call for ci tests.
     # exits with the returnCode
-    returnCode = main(verbosity=1)
+    returnCode = main(testGroup=('test', 'external'), verbosity=1, show=False)
```

Comment:
can we run this as two lines, with returnCode aggregated, so that all the `Test` are run first and then `TestExternal` after that.  In part so we can see the timing differences between the two.  I've been trying to get Test to run faster, and want to make sure we optimize the same with TestExternal

Reply:
The test controller would end up loading the doctests twice. (Or, do you only want doctests run with `Test` vs `TestExternal`? If so, that's not too difficult, but just checking if you want to go that effort.)

Reply:
Done!

```
Ran 4684 tests in 261.671s

Ran 58 tests in 30.455s
```

But this will be somewhat variable based on the nondeterministic tests.